cfe_internal/enterprise/mission_portal.cf

```diff
@@ -43,6 +67,9 @@ bundle agent cfe_internal_enterprise_mission_portal_apache
 
   methods:
 
+      "apachectl patched for resilient stop " -> { "ENT-8823" }
+        usebundle => apachectl_patched_for_upgrade;
```

Comment:
@Lex-2008 I know the ticket says:

> CFEngine versions where this should be done are probably these:
cfengine_3_18_0|cfengine_3_18_1|
cfengine_3_15.!(cfengine_3_15_6|cfengine_3_15_7)|
cfengine_3_19|cfengine_3_12|cfengine_3_10|cfengine_3_7|
cfengine_3_17|cfengine_3_16|cfengine_3_14|cfengine_3_13|cfengine_3_11|cfengine_3_9|cfengine_3_8|cfengine_3_6|cfengine_3_5

But, my logic is that all versions need this. If we ship a patched apachectl in the packages from 3.20.0, 3.18.2, and 3.15.6 forward the rendered template here should match that identically resulting in a no-op, promise kept. If they differ, of course this one will be rendered.

And once 3.21.0 is the oldest supported version, we should be able to redact this from policy and rely on the fact that the patched apachectl is in place from packages 3.20.0, 3.18.2, or 3.15.6 or later.

Reply:
Assuming people won't run newer versions of CFEngine that might have further improved versions of *apachectl* with this version of MPF.

Reply:
Or assuming that we update apachectl in the MPF as well.